packages/twenty-front/src/modules/settings/accounts/components/SettingsAccountsBlocklistTable.tsx

```diff
@@ -16,21 +16,38 @@ const StyledTable = styled(Table)`
   margin-top: ${({ theme }) => theme.spacing(4)};
 `;
 
-const StyledTableBody = styled(TableBody)`
-  border-bottom: 1px solid ${({ theme }) => theme.border.color.light};
+const StyledTableHeader = styled(TableHeader)`
+  width: 100%;
+  @media (max-width: ${MOBILE_VIEWPORT}px) {
+    white-space: pre;
+    margin-left: ${({ theme }) => theme.spacing(6)};
+  }
+
+  @media (min-width: 360px) and (max-width: 390px) {
```

Comment:
have these 360 and 390px limit been discussed with design
As far as I know we only have the MOBILE_VIEWPORT limit on the project right now.

I'm not against introducing behaviors for 360px but we need to have a project-wide approach in this case.
Can we revert this change and get the input of @Bonapara on the overall approach?

Reply:
@charlesBochet sure, I'll revert the min width code blocks